src/components/MazeMap/MazeMap.tsx

```diff
@@ -131,17 +133,74 @@ const MazeMap = (props: MazeMapProps) => {
   const prepareMap = () => {
     if (window.Mazemap) {
       const map = new window.Mazemap.Map(mapOptions);
-      map.on('load', () => {
+      map.on("load", () => {
         if (props.controls) {
           map.addControl(new window.Mazemap.mapboxgl.NavigationControl());
         }
+        if (props.hideIcons && (!props.hideAll)) {
+          let style = map.getStyle();
+
+          // layer 143 room icons
```

Comment:
could you change the name of these vars to match what they represent? eg. `let roomIcons = style.layers[143]`